src/main/js/SurveyPage.js

```diff
@@ -0,0 +1,575 @@
+                                                            updateAnswer(i, { numberAnswer : parseInt(e.target.value) || 0 });
+                                                        }
+                                                    }
+                                                }
+                                            />
+                                        </div>
+                                    );
+                                case qType.DROPDOWN:
+                                    return (
+                                        <div className="qq-app mv" key={i}>
+                                            <div>{`Question ${i + 1} - Dropdown`}</div>
+                                            <TextField
+                                                className="qq-app mv"
+                                                value={userSurvey.questions[i].question}
+                                                variant="outlined"
+                                                label="Question"
+                                                size="small"
+                                            />
+                                            <FormControl className="qq-app mv qq-app__qtype_select">
+                                                <InputLabel id={`label_${q.question}`}>{userSurvey.questions[i].question}</InputLabel>
+                                                <Select
+                                                    labelId={`label_${q.question}`}
+                                                    value={userSurvey.questions[i].stringAnswer}
+                                                    onChange={
+                                                        e => {
+                                                            if (!userSurvey.closed) {
+                                                                updateAnswer(i, { stringAnswer : e.target.value });
+                                                            }
+                                                        }
+                                                    }
+                                                >
+                                                    {q.options.map(option => (
+                                                        <MenuItem key={option} value={option}>{option}</MenuItem>
+                                                    ))}
+                                                </Select>
+                                            </FormControl>
+
+                                        </div>
+                                    );
+                                default:
+                                    console.log(`[WARNING] Unknown question type "${q.question}"`)
+                                    return (<div />);
+                            };
+                        })}
+                    </div>
+                </div>
+                <div>
```

Comment:
When I test master the `console` title overlaps with the above div after content is added to the console. Check if that is true for you and add space on top if necessary.

Reply:
This should be fixed when the pages are separated, as we will likely abstract the console to its own component. This is also not displayed in production.